libcontainer/specconv/spec_linux_test.go

```diff
@@ -546,6 +546,29 @@ func TestSpecconvExampleValidate(t *testing.T) {
 	}
 }
 
+func TestSpecconvNoLinuxSection(t *testing.T) {
+	spec := Example()
+	spec.Root.Path = "/"
+	spec.Linux = nil
+	spec.Hostname = ""
+
+	opts := &CreateOpts{
+		CgroupName:       "ContainerID",
+		UseSystemdCgroup: false,
```

Comment:
no need to specify that

Reply:
Removed UseSystemdCgroup

Reply:
I can still see it.

Reply:
Sorry. I think it's really gone this time.